app/lib/config/desktop.dart

```diff
@@ -122,6 +125,7 @@ class _DesktopSupportState extends State<DesktopSupport>
   void onTrayMenuItemClick(MenuItem menuItem) async {
     if (menuItem.key == 'exit_app') {
       _log.info('exit app');
+      await trayManager.destroy();
       await windowManager.destroy();
```

Comment:
did you test this on other platforms like Linux and windows as well? Would be unfortunate if a change like this means we can't exit the desktop anymore on any target...

Reply:
Yeah, you discussed the important thing.
I missed that this PR should be tested on various platforms.
Above videos were created on windows.
All to do is test on linux and macos.
@gnunicorn Can you please build it on them for me?